AI_ML/SEGMENTATION/DEEPLAB_V3/DEEPLAB_V3.py

```diff
@@ -11,26 +11,26 @@
     ]
 )
 def DEEPLAB_V3(default: Image) -> Image:
-    """The DEEPLAB_V3 node returns a segmentation mask from an input image
-    in a dataframe. The input image is expected to be a DataContainer of type
-    "image". The output is a DataContainer of type "image" with the same
-    dimensions as the input image, but with the red, green, and blue channels
-    replaced with the segmentation mask.
+    """
+    The DEEPLAB_V3 node returns a segmentation mask from an input image in a dataframe.
+    
+    The input image is expected to be a DataContainer of an "image" type.
+    
+    The output is a DataContainer of an "image" type with the same dimensions as the input image, but with the red, green, and blue channels replaced with the segmentation mask.
 
     Parameters
     ----------
     None
 
-    Returns:
-    --------
-    DataContainer:
+    Returns
+    -------
+    DataContainer
         A DataContainer with the following fields:
```

Comment:
Don't even need a description for this, all this does is explain what `Image` is